schema/snapcraft.json

```diff
@@ -945,6 +945,7 @@
                             "items": {
                                 "type": "string",
                                 "enum": [
+                                    "enable-patchelf",
```

Comment:
as jedi master of jsonschema would you mind adding a conflict for enable-patchelf and no-patchelf?

Reply:
As jedi master of jsonschema, I declare this impossible at least with how much I (as a jedi) have forgotten. List items are validated per-item, not as a set. The only way I see this working is if we made build attributes object properties instead of just a list of strings. So I punted and handled with a proper exception instead, seem okay?